index.html

```diff
@@ -928,7 +929,8 @@
           Interface <a><code>PresentationRequest</code></a>
         </h3>
         <pre class="idl">
-          [Constructor(DOMString url)]
+          [Constructor(DOMString url),
+           Constructor(DOMString[] urls)]
```

Comment:
Sorry for the delayed reply, was on vacation last week.  The dual constructor form should be backwards compatible, although do I assume an implementation will be able to overload a single-argument constructor in this way?  For example code generated from the .idl would need to examine the argument type to dispatch to an appropriate function.

I'm not opposed to using a union type instead if there's a preference.